### PR TITLE
#65: Change greedy algorithm save file to include actual values of parameters

### LINKED ISSUE #65: Change greedy algorithm save file to include actual values of parameters
The greedy algorithm should save to file the actual values of the parameters it has run the FOM for. Right now, it just saves the sample index.

### PR DESCRIPTION
**Pull request by cwschilly**
Resolves #65

### CODE REVIEW

**Comment by cwschilly** [comment]
Is it a strong enough test to only test the shape of the resulting array?

File: romtools/workflows/greedy/run_greedy.py [hunk 1]
```diff
@@ -294,7 +294,8 @@ def reg(val):
                 training_samples=training_samples,
                 fom_time=fom_time,
                 rom_time=rom_time,
-                basis_time=basis_time)
+                basis_time=basis_time,
+                parameter_samples=parameter_samples)
```
Comment (fnrizzi) [inline romtools/workflows/greedy/run_greedy.py:298]: I don't think this is the right place , Eric was pointing to line 202. This is writing the sample inside the stats at the end , while we want to write them "on the fly".

**Review by fnrizzi — CHANGES_REQUESTED** [review]
The implementation is not right. Please see my comment and also Eric comment about where to change the code. 

For the test, we need a test that checks sample at every iteration.

**Comment by fnrizzi** [comment]
> Is it a strong enough test to only test the shape of the resulting array?

Please see my comments

File: romtools/workflows/greedy/run_greedy.py (hunk 1 above)
Comment (cwschilly) [reply romtools/workflows/greedy/run_greedy.py]: Resolved

**Pull request merged**